content/services.md

```diff
@@ -0,0 +1,75 @@
+
+Meanwhile, our larger PA systems can be used in a number of different configurations to provide the sound you want for the venue you’ve got, as loud as you want it (within reason!). We use both the Nexo Alpha and Alpha E lines of PA to find the best configuration for your event.
+
+To complement our Nexo PA, we have invested in a d&b T-10 Line Array system, which can also be used in a point source configuration. This allows us greater flexibility than ever before in getting the perfect configuration for every event, in every venue.
+
+Our amplifier stock is perfectly suited to our PA stock, many featuring in-house tweaks coupled with custom cabling to allow for every circumstance. TEC can meet any requirements in terms of noise, speaker arrangement and power availability.
+
+When receiving a sound package from TEC, the charge for the amplifiers are included with the speakers themselves as a combined cost. Professional setup and control is also included with the cost of the equipment. It's what we do.
+
+### Lighting Services
+
+{{< intext_image src="/imgs/services/diwali.jpg" width="100%" alt="Haze and lights in a large space">}}
+
+TEC can provide almost any lighting configuration for your event, from a couple of spotlights to a DJ club night. We’ve done it all, and are happy to work to your specifications. We will set up, program and operate all of our lighting equipment and control desks as standard. That’s not to say you’re ignored: we’re here to be spoken to, so ask and you will receive.
+
+
+{{< intext_image src="/imgs/services/generic.jpg" width="55%" float="right" alt="Silhouetted person focusing a light">}}
+
+#### Generic Lighting
+
+These are the standard dimmable on/off kind of lights that are great for spots and stage washes. They are straightforward to set up and can have colour and patterned gobo filters added for effective mood lighting.
+
+The colour and position can’t change during the event, but they are perfect at what they do and few lighting events work without them.
+
+{{< intext_image src="/imgs/services/intel.jpg" width="55%" float="right" alt="Lights with beams in haze behind a DJ deck">}}
+
+#### Intelligent Lighting
+
+For when you want to take things up a notch. From changing colours, spinning around and strobing, our intelligent lighting stock can give you that look to make your event stand out from the crowd.
+
+Our MAC 250 Washes give you the freedom to move your stage wash wherever you want, and change the color, during the event. Our workhorse spotlight fixtures, the MAC 250 Kryptons, also have a variety of colours available, and a wide selection of gobos and good beam control. If these aren't enough we have Robe DL4Ss for bright and precise control with a full spectrum of RGB color. If it’s just effective mood colouring that you’re after, our StudioCob LED fixtures are the ideal choice, while a Technobeam or four can make a club or dance night come to life.
```

Comment:
Can we make the lighting department sell the good lights rather than the knackered old broken ones...?